docs/devel/developer-guides/vagrant.md

```diff
@@ -34,45 +34,74 @@ Documentation for other releases can be found at
 
 ## Getting started with Vagrant
 
-Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/develop on your local machine (Linux, Mac OS X).
+Running kubernetes with Vagrant (and VirtualBox) is an easy way to
+run/test/develop on your local machine (Linux, Mac OS X).
 
 ### Prerequisites
 
-1. Install latest version >= 1.6.2 of vagrant from http://www.vagrantup.com/downloads.html
+1. Install latest version >= 1.8.1 of vagrant from
+http://www.vagrantup.com/downloads.html
+
+
 2. Install one of:
-   1. The latest version of Virtual Box from https://www.virtualbox.org/wiki/Downloads
-   2. [VMWare Fusion](https://www.vmware.com/products/fusion/) version 5 or greater as well as the appropriate [Vagrant VMWare Fusion provider](https://www.vagrantup.com/vmware)
-   3. [VMWare Workstation](https://www.vmware.com/products/workstation/) version 9 or greater as well as the [Vagrant VMWare Workstation provider](https://www.vagrantup.com/vmware)
-   4. [Parallels Desktop](https://www.parallels.com/products/desktop/) version 9 or greater as well as the [Vagrant Parallels provider](https://parallels.github.io/vagrant-parallels/)
+
+   1. The latest version of Virtual Box from
```

Comment:
@vishh -we give so many options because people have asked for them and added them.